app/models/missing_basket.rb

```diff
@@ -2,4 +2,8 @@ class MissingBasket
   def line_items
     LineItem.none
   end
+
+  def to_partial_path
+    'empty_basket'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.